### PR TITLE
Dpe 2259 Footer gedöns 

### PR DESCRIPTION
**Pull request by StefanVesper**

### CODE REVIEW

File: src/assets/fixtures/footer/footer_hessenschau.json
Comment (Paul-Atreidis) [inline src/assets/fixtures/footer/footer_hessenschau.json:1]: TODO Missing link object, need to refactor!

File: src/assets/fixtures/footer/footer_hr1.json
Comment (Paul-Atreidis) [inline src/assets/fixtures/footer/footer_hr1.json:1]: TODO Missing link object, need to refactor!

File: src/assets/fixtures/footer/footer_hr2.json
Comment (Paul-Atreidis) [inline src/assets/fixtures/footer/footer_hr2.json:1]: TODO Missing link object, need to refactor!

File: src/assets/fixtures/footer/footer_hr3.json
Comment (Paul-Atreidis) [inline src/assets/fixtures/footer/footer_hr3.json:1]: TODO Missing link object, need to refactor!

File: src/assets/fixtures/footer/footer_hr4.json
Comment (Paul-Atreidis) [inline src/assets/fixtures/footer/footer_hr4.json:1]: TODO Missing link object, need to refactor!

File: src/assets/fixtures/footer/footer_hrinfo.json
Comment (Paul-Atreidis) [inline src/assets/fixtures/footer/footer_hrinfo.json:1]: TODO Missing link object, need to refactor!

File: src/assets/fixtures/footer/footer_youfm.json
Comment (Paul-Atreidis) [inline src/assets/fixtures/footer/footer_youfm.json:1]: TODO Missing link object, need to refactor!

File: src/assets/fixtures/footer/footer_hessenschau.json
Comment (szuelch) [reply src/assets/fixtures/footer/footer_hessenschau.json:1]: Habe mich da mit Steve abgestimmt. Das stimmt jetzt eigentlich so.

**Review by Paul-Atreidis — APPROVED** [review]
All fine

**Review by szuelch — APPROVED** [review]

**Pull request merged**